src/models/encounter/useDeleteEncounter.js

```diff
@@ -36,13 +37,24 @@ export default function useDeleteEncounter() {
         setError(null);
         return true;
       }
-
       setError(formatError(deleteResponse));
       setSuccess(false);
       return false;
     } catch (postError) {
+      const vulnerableIndividualGuid =
+        postError?.response?.data?.vulnerableIndividualGuid;
+      const vulnerableSightingGuid =
+        postError?.response?.data?.vulnerableSightingGuid;
+      const vulnerableOject = vulnerableSightingGuid
```

Comment:
Typo in the variable name.
Because the correct spelling would shadow your component state's variable, maybe don't assign to a separate variable at all? `setVulnerableObject(vulnerableSightingGuid ?? vulnerableIndividualGuid)`.

Of course, you could just pick a different name too.

Reply:
Done.